packages/coin98-wallet/src/lib/coin98-wallet.ts

```diff
@@ -195,7 +198,7 @@ export const setupCoin98Wallet = ({
         iconUrl,
         downloadUrl:
           "https://chrome.google.com/webstore/detail/coin98-wallet/aeachknmefphepccionboohckonoeemg",
-        deprecated: false,
+        deprecated,
```

Comment:
Missing deprecated: deprecated ?

Reply:
We set the value of  `deprecated` to false by default see [line 181](https://github.com/near/wallet-selector/blob/release-improvements/packages/coin98-wallet/src/lib/coin98-wallet.ts#L181) we assume here the wallet is not deprecated.

This can be configured/changed by dApp developers when they call the setup function of any wallet during the initialization of Wallet Selector:

```ts
const _selector = await setupWalletSelector({
  network: "testnet",
  debug: true,
  modules: [setupCoin98Wallet({ deprecated: true })],
});
```

Reply:
The syntax here is just a shorthand JS syntax.

```ts
const deprecated = false;

const someObject = {
  deprecated: deprecated
}

const someShorthandObject = {
  deprecated
}

console.log(someObject.deprecated) // logs false
console.log(someShorthandObject.deprecated) // logs false

```